homeassistant/backup_restore.py

```diff
@@ -0,0 +1,113 @@
+"""Home Assistant module to handle restoring backups."""
```

Comment:
So while the `.HA_RESTORE` file _can_ be manually added, I dont think that is something we should document.
So the documentation requirement of this feature, IMO should not be triggered until this is implemented in the UI, or a service call is added.